spec/30-processing-model.md

```diff
@@ -0,0 +1,52 @@
+# Processing Model
+
+This section provides a step-by-step description of the behavior of a tracing implementation - also called tracer -  when a request is received. This description can be used as a reference when implementing a Trace Context compliant tracing system, middleware or cloud service.
+
+## Processing Model for Working with Trace Context
+
+The processing model describes the behaviour of a tracer which forwards **and** modifies
+Trace Context headers.
+
+1. The tracer receives a `traceparent` header and may receive a `tracestate` header.
```

Comment:
Valid point. I think I will add a section for starting a trace separately. This still adds the question on what to do when receiving a trace-state without a trace-parent. It can be that there was a proxy or the request is broken.

Reply:
Right. My understanding it that `traceparent` is required. An orphaned `tracestate` could/should be thrown away - just like if the `tracestate` is accompanied with an invalid `traceparent`.